### ISSUE TITLE
Modify Device State

### ISSUE THREAD

**Issue by hspak** [awslabs/aws-device-farm-jenkins-plugin#27]
Is it possible or will it ever be possible to modify the device state through the plugin? I'd like to upload dependencies, but I haven't found any way.

**Comment by appwiz** [comment]
Hi @hspak, could you provide more detail about your use case? If you are looking to upload additional applications as dependencies, you can specify a [list of AuxiliaryApps](http://docs.aws.amazon.com/devicefarm/latest/APIReference/API_ScheduleRunConfiguration.html#devicefarm-Type-ScheduleRunConfiguration-AuxiliaryApps) in your ScheduleRun request through the plugin.

**Comment by hspak** [comment]
Yes that was exactly what I was looking for, thanks!

Are there any plans to expose [these](https://github.com/awslabs/aws-device-farm-jenkins-plugin/blob/8f215e59b616bc78ff1c95fef3d2b4a31ee22b83/src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmRecorder.java#L358) through the web UI? This is what caused my confusion.

**Comment by appwiz** [comment]
@hspak yes, it is in our backlog but unfortunately no ETA at this time. If you want to put together the UI and submit a pull request, we are happy to take a look at it.
